pages/treeherder.py

```diff
@@ -59,10 +75,13 @@ def open_perfherder_page(self):
         from perfherder import PerfherderPage
         return PerfherderPage(self.base_url, self.selenium).wait_for_page_to_load()
 
-    def open_single_resultset(self):
```

Comment:
I don't think you meant to remove this - and it probably explains @stephendonner's issues when running the tests.